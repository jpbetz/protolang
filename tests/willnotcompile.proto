def willnotcompile(x( 1;
