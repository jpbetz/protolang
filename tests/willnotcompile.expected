"<stdin>" (line 1, column 21):
unexpected "("
expecting identifier or ")"
